code/modules/admin/view_variables/reference_tracking.dm

```diff
@@ -0,0 +1,46 @@
+/world/proc/enable_reference_tracking()
+	var/extools = world.GetConfig("env", "EXTOOLS_DLL") || (world.system_type == MS_WINDOWS ? "./byond-extools.dll" : "./libbyond-extools.so")
+	if (fexists(extools))
+		call(extools, "ref_tracking_initialize")()
+
+/proc/get_back_references(datum/D)
+	CRASH("/proc/get_back_references not hooked by extools, reference tracking will not function!")
+
+/proc/get_forward_references(datum/D)
+	CRASH("/proc/get_forward_references not hooked by extools, reference tracking will not function!")
+
+/proc/clear_references(datum/D)
+	return
+
+/datum/Del()
```

Comment:
We absolutely cannot do this under any circumstance, ever.

If `Del` is even *defined* on an object, it dramatically increases the time/cost it takes to delete.

The fact this is defined at the `/datum` level means that now a ton of things are going to get incredibly more expensive to delete----and some things are just never going to GC (like mobs).

This is a great PR, but we can never, ever ever define `Del`.

Reply:
Just needs an ifdef over the whole file. Same as as our current soft ref tracker.